playbooks/roles/plugins/ansible/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+- name: 'Install ansible'
+  yum:
+    name: ansible
+    state: present
+  tags:
+    - installation
+
+- name: 'Install ansible plugin'
+  shell: >
+    foreman-installer
+    --enable-foreman-plugin-ansible
+    --enable-foreman-proxy-plugin-ansible
```

Comment:
On most Katello installations this will likely fail out with no re-course due to the system checks such as requiring 8GB of memory meant to protect users in production but not absolutely required for dev or testing.